packages/dustkit/src/bridge/schemas.ts

```diff
@@ -61,4 +61,17 @@ export type ClientRpcSchema = RpcSchema.From<
         z: number;
       };
     }
+  | {
+      Request: {
+        method: "setBlueprint";
+        params: {
+          objectTypeId: number;
+          x: number;
+          y: number;
+          z: number;
+          rotation: number;
```

Comment:
is this what its called in the contracts? like in the blueprint schema? we should match whatever its called there

Reply:
good point, it's actually `orientation`, updated